src/app/components/Header/ScriptLink/index.tsx

```diff
@@ -63,13 +63,12 @@ const ScriptLink = ({ scriptSwitchId = '' }) => {
   const { enabled: scriptLinkEnabled } = useToggle('scriptLink');
   const { enabled: variantCookieEnabled } = useToggle('variantCookie');
 
+  if (!scriptLinkEnabled || isNextJs) return null;
```

Comment:
Think this is maybe a merge conflict from yesterday, resulting in it being moved above the comment below.